src/constants/web3-providers.ts

```diff
@@ -0,0 +1,6 @@
+export const web3Providers: Record<number, { url: string; token: string }> = {
```

Comment:
Move to blockchains/tokens/web3.providers.ts

Reply:
Marked as Resolved due to outdated.